src/sentry/sentry_metrics/querying/eap/protobufs/AggregateBucket_pb2.pyi

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
we should not be checking in generated files

Reply:
I would recommend making a python package client library and adding that as a dependency rather than vendoring this in like this

Reply:
this is a short term thing, we will migrate to the protos repo when things are more stable there

Reply:
let's do the right thing first rather than hacking this together -- no need to clutter the repo and be risky

Reply:
the tooling doesn't exist yet, it's a chicken and egg problem right now because nobody uses protobuf, so there is no tooling, so we can't use protobuf 😿

Reply:
so that should tell you that protobuf is not a viable technology to be used at sentry _yet_ and you should use a supported approach rather than taking a risky one

Reply:
The tooling will exist *soon*. I'm actively working on it and hope to have something later this week. I agree with @asottile-sentry about not making a mess for ourselves in the short-term.

Reply:
Thanks, @markstory. So this whole topic will resolve itself this week?